DSE_6.8_Release_Notes.md

```diff
@@ -47,6 +70,7 @@ If you're developing applications, please refer to the [Java Driver documentatio
 
 ## 6.8.27 DSE Graph
 * Fixed GraphOLAP spark connection problem in the multi-network cloud environments when GPFS is not used. (DSP-22707)
```

Comment:
I know it's a previous release but it seems sensible to include the full name GossipingPropertyFileSnitch as well as the abbreviation, for easier searchability.

Reply:
Indeed. But people usually use `GPFS` abbreviation.
See history of the original ticket (DSP-22707) for "why sometimes abbreviations are better and we should check emails".

Reply:
after finding `General Parallel File System` I give up and vote to stick to the abbreviation.

Reply:
Why not write:

> Fixed GraphOLAP spark connection problem in the multi-network cloud environments when GossipingPropertyFileSnitch (GPFS) is not used. (DSP-22707)

?